tools/get_channel.py

```diff
@@ -0,0 +1,144 @@
+#############################################################################
+# get_channel.py                                                            #
+# Script to load (T)ELLIE channel information from couchDB                  #
+# 1. Prints HW info                                                         #
+# 2. Plots calibration (tuning) curves                                      #
+#                                                                           #
+# Date created: 07/May/2019                                                 #
+# Author: M Rigan <m.rigan@sussex.ac.uk> : New file                         #
+#############################################################################
+
+# imports
+from orca_side import tellie_database
+import argparse
+import sys
+import json
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import numpy as np
+
+# text formatting
+matplotlib.rcParams.update({'font.size': 13})
+
+# set up access to database
+database = tellie_database.TellieDatabase('http://couch.snopl.us', 'telliedb')
+
+# load channel data from couchDBs
+def get_channel(channel):
+    # key for query
+    key = [ int(channel), [ 10870, 2147483647 ], 0, -2 ]
```

Comment:
Is this a range? Should perhaps belong in a config file too?

Reply:
this spans the start of the tellie use and end of tellie tables, it's not an issue of this small piece of code but deeper sno+ setting, will leave like this for now (should not ever change)